launch/c1t_bringup_launch.xml

```diff
@@ -12,8 +12,11 @@
   </include>
 
   <!--Launch files exclusive to Turtlebot simulator-->
+  <include file="$(find-pkg-share turtlebot3_gazebo)/launch/turtlebot3_world.launch.py" if="$(eval '\'$(var vehicle)\' == \'turtlebot\'')"/>
+
   <node pkg="rviz2" exec="rviz2" name="rviz" args="--display-config $(find-pkg-share c1t_bringup)/rviz/nav2_route.rviz" if="$(eval '\'$(var vehicle)\' == \'turtlebot\'')"/>
 
+
```

Comment:
Unnecessary space? Otherwise looks fine as long as it works which I assume you have tested

Reply:
Just fixed the space, and yes I confirmed this works